
# Exercício 1
## Parte 1
Crie 3 variáveis:

nomePrimeiro, nomeMeio, nomeUltimo

Atribua um valor para cada uma delas e imprima a seguinte frase para o usuário:

"Gabriel Gonçalves de Briro, seja muito bem vindo a DNC"

Não precisa usar input( )

In [28]:
nomePrimeiro= 'Gabriel '
nomeMeio= 'Gonçalves de '
nomeUltimo= 'Brito'

print(nomePrimeiro + nomeMeio + nomeUltimo + ', seja bem vindo a DNC')

Gabriel Gonçalves de Brito, seja bem vindo a DNC


## Parte 2
Agora peça para o usuário digitar as 3 variáveis antes de mostrar a mesma frase da parte 1.

Resultado esperado

In [29]:
nomePrimeiro=input('Qual o seu primeiro nome?\n')
nomeMeio=input('Qual o seu nome do meio?\n')
nomeUltimo=input('Qual o seu ultimo nome?\n')

print(nomePrimeiro, nomeMeio, nomeUltimo + ', seja muito bem vindoa DNC')

Gabriel Gonçalves de Brito, seja muito bem vindoa DNC


# Exercício 2
#### 1. Vamos criar nosso primeiro script 💪, que:

Armazene em uma variavel o nome do cliente, a variável precisa se chamar cliente_nome e o nome do cliente você pode usar sua criatividade.

Crie uma variável que armazene a idade do cliente, a variável precisa se chamar cliente_idade.

Exiba na tela as 2 variáveis e apenas uma variável por linha. Ex: "O nome do Cliente é: Nathan Pereira".

In [24]:
cliente_nome='Gabriel Gonçalves de brito'
cliente_idade= 25

print('Nome do cliente:', cliente_nome)
print('A idade do cliente:', cliente_idade)

Nome do cliente: Gabriel Gonçalves de brito
A idade do cliente: 25


#### 2. Agora que você já criou seu primeiro script vamos aumentar a dificuldade!
Agora é o cliente que vai colocar as informações! Crie uma variável que receba como input o nome e a idade do cliente!
Para ajudar o usuário exiba, essas variáveis na tela.

In [3]:
cliente= input('Qual seu nome?\n')
idade= input('Qual a sua idade?\n')

print('O nome do clinte é:', cliente)
print('A idade do cliente é:', idade)

O nome do clinte é: Gabriel Gonçalves de Brito
A idade do cliente é: 25


#### 3. Vamos brincar com mais variáveis agora?!
Crie uma variável do tipo FLOAT que vai armazenar o valor do produto, que receberá o valor de 25.30!
Crie uma variável do tipo BOOL que vai armazenar se ele é ou não um cliente premium, recebendo o valor de True.
Agora exiba essas variáveis na tela.
Exiba também o tipo dessas variáveis.

In [20]:
produto_valor=25.30
cliente_premium=True

print('O valor do produto é:', produto_valor)
print ('O tipo da variável produto-valor é:', type( produto_valor))
print('O cliente é premium?', cliente_premium)
print('O tipo da variável cliemte_premium é:', type(cliente_premium))
print()

print('o valor do produto é: ,produto_valor', produto_valor, type (produto_valor))
print('o cliente é premium?', cliente_premium, type(cliente_premium))

O valor do produto é: 25.3
O tipo da variável produto-valor é: <class 'float'>
O cliente é premium? True
O tipo da variável cliemte_premium é: <class 'bool'>

o valor do produto é: ,produto_valor 25.3 <class 'float'>
o cliente é premium? True <class 'bool'>


### tipos de dados

*str:* cadeia de caracteres= ']~´[@', '7', 'Gabriel'

*int:* inteiro= 5, 10, 25

*float:* real= 5.3, 25.3

*bool:* booleanos= true, false


### funções

**print()=** imprimr uma frase

**type()=** verifica o tipo de dado de uma variável

**input()=** pede pro usuário fornecer um valor (str)

# plote as variáveis a baixo:
## a = 245
## b = 2.835895

In [41]:
a = 245
b = 2.835895

print('| {0:10} | e | {1:10.3f} |'.format(a, b))

|        245 | e |      2.836 |


### Fui numa hamburgueria e comprei 2 lanches de 10,90 cada, 3 refris de 4,50 cada e 1 batata por 8,45.
### O pagamento foi no dinheiro com notas de 10 reais e 5 reais. perguntas
1. Qual foi o valor total da conta?
2. Quantas notas de 10 reais eu usei?
3. Quantas notas de 5 reais eu usei?
4. Quanto voltou de troco para mim em notas?
5. Qunato voltou de troco para mim em moedas?

In [38]:
qunatidadeLanche=2
quantidadeRefri=3
quantidadebatata=1
valorlanche=10.9
valorRefri=4.5
valorBatata=8.45

valorTotal= qunatidadeLanche * valorlanche + quantidadeRefri * valorRefri + quantidadebatata * valorBatata

print('O valor da conta foi de: R$ {}'.format(valorTotal))

notas10=valorTotal // 10

print('Você tera que utilizar: {} notas de R$ 10,00'.format(int(notas10)))

valorFalta = valorTotal - notas10 * 10 
notas5 = valorFalta // 5 + 1
valorFalta= valorFalta - notas5

print('Você utilizara {} notas de R$ 5,00'. format (int(notas5)))

valorFalta = valorFalta - notas5
notas2= valorFalta // 2


print('Eu recebi {} notas de R$ 2,00'.format(int(notas2)))

valorMoeda= valorFalta - 0

print('Vou receber R$ {} em moedas'.format(valorMoeda))


O valor da conta foi de: R$ 43.75
Você tera que utilizar: 4 notas de R$ 10,00
Você utilizara 1 notas de R$ 5,00
Eu recebi 0 notas de R$ 2,00
Vou receber R$ 1.75 em moedas


# Exercícios
## Exercício 1:
#### Vamos testar os comportamentos dos diferentes operadores aritméticos. Para isso vamos montar uma calculadora, eai? Bora, vamos nessa?

Criar variáveis n1 e n2 que receberão inputs do tipo INT do usuário. (não se esqueça de tratar o que está sendo recebido rsrs) Exibir o resultado da Adição dessas 2 variáveis. Exibir o resultado da Subtração dessas variáveis. Exibir o resultado da Multiplicação dessas variáveis. Exibir o resultado da Divisão dessas variáveis. Exibir o resultado do Módulo (Resto da divisão) entre essas variáveis. Exibir o resultado de uma variável Exponenciando a outra. [ ]

In [2]:
n1=int(input("igite um valor inteiro"))
n2=int(input('informe mais um valor inteiro'))

print ('n1:',n1,
'\nn2:',n2)
print (n1+n2)
print (n1-n2)
print (n1*n2)
print (n1/n2)
print (n1//n2)
print (n1%n2)

n1: 7 
n2: 5
12
2
35
1.4
1
2


## Exercício 2:
Faça um programa que o usuário digita um valor inteiro e nós mostramos toda a tabuada daquele número.

In [42]:
contador = 0
valor=int(input('informe um valor inteiro\n'))
while True:
    if contador < 10:
        contador = contador + 1
        if contador :
            print(('{} * {} = {}'.format(valor, contador, valor*contador)))
    else:
        break
print("chegamos ao fim, até a proxima")

18 * 1 = 18
18 * 2 = 36
18 * 3 = 54
18 * 4 = 72
18 * 5 = 90
18 * 6 = 108
18 * 7 = 126
18 * 8 = 144
18 * 9 = 162
18 * 10 = 180
chegamos ao fim, até a proxima


## Exercício 3:
Faça um programa no qual o usuário digite a distância percorrida por um carro e a qtd de litros de combustível gasto.
Calcule e mostre para o usuário quantos km/l o seu carro faz.

In [45]:
quilometros=int(input('Quantos km você percorreu com seu carro? \n'))
litrosConbustivel=int(input('Quantos litros de conbustível foi gastos?\n'))
km_L= quilometros / litrosConbustivel
print('Seu carro faz {:.2f}km/L'.format(km_L))

Seu carro faz 16.67km/L
